connectivity/suite.go

```diff
@@ -345,13 +350,13 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 		WithPolicy(echoIngressICMPDenyPolicyYAML). // Deny ICMP traffic from client to another client
 		WithFeatureRequirements(check.RequireFeatureEnabled(check.FeatureICMPPolicy)).
 		WithScenarios(
-			tests.PodToPod(),       // Client to echo traffic should be allowed
+			//tests.PodToPod(),       // Client to echo traffic should be allowed
```

Comment:
Why is this commented out?

Reply:
sorry, it was unintentional as part of debugging :100: